packages/core/src/utils-hoist/promisebuffer.ts

```diff
@@ -9,6 +8,8 @@ export interface PromiseBuffer<T> {
   drain(timeout?: number): PromiseLike<boolean>;
 }
 
+export const SENTRY_BUFFER_FULL_ERROR = Symbol('SentryBufferFullError');
```

Comment:
dunno: Maybe we should do `Symbol.for()` if multiple versions of the SDK end up in one application.

```suggestion
export const SENTRY_BUFFER_FULL_ERROR = Symbol.for('SentryBufferFullError');
```